src/OneExplorer/OneExplorer.ts

```diff
@@ -150,17 +150,30 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<OneNode> {
   private fileWatcher =
       vscode.workspace.createFileSystemWatcher(`**/*.{cfg,tflite,onnx,circle,tvn}`);
 
+  private tree: Node|undefined;
+
   constructor(private workspaceRoot: vscode.Uri) {
     const fileWatchersEvents =
         [this.fileWatcher.onDidCreate, this.fileWatcher.onDidChange, this.fileWatcher.onDidDelete];
 
     for (let event of fileWatchersEvents) {
-      event(() => this._onDidChangeTreeData.fire());
+      event(() => this.refresh());
```

Comment:
This change is required. 

FYI, what `_onDidChangeTreeData.fire()` does:
- it's an event api given by vscode. (vscode.TreeViewProvider)
- it calls `getTree()`, which will remain return the same tree if we don't reset `this.tree`.
- NOW, `this.refresh()` does what we want: reset `this.tree` and fires